src/test/java/org/jabref/gui/externalfiles/ImportHandlerTest.java

```diff
@@ -54,4 +103,118 @@ void handleBibTeXData() {
 
         assertEquals(List.of(expected), bibEntries.stream().toList());
     }
+
+    @Test
+    void cleanUpEntryTest() {
+        BibEntry entry = new BibEntry().withField(StandardField.AUTHOR, "Clear Author");
+        BibEntry cleanedEntry = importHandler.cleanUpEntry(bibDatabaseContext, entry);
+
+        assertEquals("Clear Author", cleanedEntry.getField(StandardField.AUTHOR).orElse(""));
```

Comment:
One can use `Optional` also in assertEquals

```suggestion
        assertEquals(Optional.of("Clear Author"), cleanedEntry.getField(StandardField.AUTHOR));
```